docs/compendium/public_share.md

```diff
@@ -52,10 +70,93 @@ Depending on the file structure, the public share contents are treated different
 {"error":"public share host is not allowed"}
 ```
 
-## Example data
+### Example data
 
 For testing purposes you can use the following public share. It contains a few ready-to-use compendia found in the [o2r-bagtainers](https://github.com/o2r-project/o2r-bagtainers) project:
 
 `https://uni-muenster.sciebo.de/index.php/s/7EoWgjLSFVV89AO`
 
-For more configuration details, see the project's README file.
+## Zenodo
+
+### Body parameters for creating compendium from a Zenodo record
+
+- `share_url` - The link to the zenodo record (optional). May be a link to https://zenodo.org or https://doi.org
+- `doi` - A [DOI](https://doi.org) resolving to the zenodo record (optional)
+- `zenodo_record_id` - The ID of the zenodo record (optional)
+- `content_type` - Form of archive. One of the following (required):
+    - `compendium_v1` compendium in Bagtainer format
+    - `workspace` - _[NOT IMPLEMENTED]_ - formless workspace
+- `filename` - Filename of your compendium. For now, only zip-files are supported. (optional)
+    - if no `filename` is provided the first zip file is selected
+    - multiple files are currently not supported
+
+There must at least one url parameter that resolves to a zenodo record. I.e. one of the following:
+
+- `share_url`
+- `doi`
+- `zenodo_record_id`
+
+### Examples
+
+1. Zenodo Record URL (with optional filename parameter)
+
+```bash
+curl -d "content_type=compendium_v1" \
+    -d "zenodo_url=https://sandbox.zenodo.org/record/69114"  \
+    -d "filename=metatainer.zip" \
+    --cookie "connect.sid=<code string here>" \
+     http://…:8088/api/v2/compendium
+```
+
+2. DOI
+
+```bash
+curl -d "content_type=compendium_v1" \
+    -d "doi=10.5072/zenodo.69114"  \
+    --cookie "connect.sid=<code string here>" \
+     http://…:8088/api/v2/compendium
+```
+
+3. Zenodo Record ID
```

Comment:
Please add a note here that if only the record ID is provided, then a default of Zenodo and Zenodo sandbox is used as selected by the API maintainer.